express-api/src/controllers/projects/projectsController.ts

```diff
@@ -0,0 +1,325 @@
+   * }]
+   */
+  return stubResponse(res);
+};
+
+/**
+ * @description Delete disposal project by either the numeric id or projectNumber.
+ * @param {Request}     req Incoming request.
+ * @param {Response}    res Outgoing response.
+ * @returns {Response}      A 200 status with the deleted project.
+ */
+export const deleteDisposalProject = async (req: Request, res: Response) => {
+  /**
+   * #swagger.tags = ['Projects']
+   * #swagger.description = 'Delete the project for the specified id.'
+   * #swagger.security = [{
+   *   "bearerAuth" : []
+   * }]
+   */
+  return stubResponse(res);
+};
+
+/**
+ * @description Add disposal project.
+ * @param {Request}     req Incoming request.
+ * @param {Response}    res Outgoing response.
+ * @returns {Response}      A 200 status with the new project.
+ */
+export const addDisposalProject = async (req: Request, res: Response) => {
+  /**
+   * #swagger.tags = ['Projects']
+   * #swagger.description = 'Add a new disposal project.'
+   * #swagger.security = [{
+   *   "bearerAuth" : []
+   * }]
+   */
+  return stubResponse(res);
+};
+
+/**
+ * @description Request a status change for the project with the specified id..
+ * @param {Request}     req Incoming request.
+ * @param {Response}    res Outgoing response.
+ * @returns {Response}      A 200 status with the requested project.
+ */
+export const requestProjectStatusChange = async (req: Request, res: Response) => {
+  /**
+   * #swagger.tags = ['Projects']
+   * #swagger.description = 'Request a status change for the project with the specified id.'
+   * #swagger.security = [{
+   *   "bearerAuth" : []
+   * }]
+   */
+  return stubResponse(res);
+};
+
+/**
+ * @description Get a report for the specified project.
+ * @param {Request}     req Incoming request.
+ * @param {Response}    res Outgoing response.
+ * @returns {Response}      A 200 status with the requested report.
+ */
+export const getProjectReport = async (req: Request, res: Response) => {
+  /**
+   * #swagger.tags = ['Projects']
+   * #swagger.description = 'Get the report for the project with the specified id.'
+   * #swagger.security = [{
+   *   "bearerAuth" : []
+   * }]
+   */
+  return stubResponse(res);
+};
+
+/**
+ * @description Update the report for the specified project.
+ * @param {Request}     req Incoming request.
+ * @param {Response}    res Outgoing response.
+ * @returns {Response}      A 200 status with the updated report.
+ */
+export const updateProjectReport = async (req: Request, res: Response) => {
+  /**
+   * #swagger.tags = ['Projects']
+   * #swagger.description = 'Update the report for the project with the specified id.'
+   * #swagger.security = [{
+   *   "bearerAuth" : []
+   * }]
+   */
+  return stubResponse(res);
+};
+
+/**
+ * @description Update the report for the specified project.
+ * @param {Request}     req Incoming request.
+ * @param {Response}    res Outgoing response.
+ * @returns {Response}      A 200 status with the deleted report.
+ */
```

Comment:
```suggestion
 * @description Delete the report for the specified project.
 * @param {Request}     req Incoming request.
 * @param {Response}    res Outgoing response.
 * @returns {Response}      A 200 status with the deleted report.
 */
```
I wonder if we should also consider what kind of response we want to return for deletions. 200 makes sense I guess if we're returning something, but I feel like I've seen a lot of cases in the original that could have just sent a 204 response back.

Reply:
Either way would definitely be fine, I guess it just depends on whether the frontend might be able to do something useful with the entire deleted resource or not (ie. show the user that the snapshots were deleted in addition to the requested report)